staging-testnet.yaml

```diff
@@ -29,5 +29,5 @@ elements:
 - Nodes
 - Explorer
 - ReportServer
-nixops: /nix/store/mb4rfbkliwjvcdcnqzgg4bgi45i8pb3s-nixops-1.6pre0_abcdef/bin/nixops
+nixops: /nix/store/7ckasdkad0vl5y1lpi2arlh6fipfmkv7-nixops-1.6pre0_abcdef/bin/nixops
```

Comment:
This should have a corresponding change in default.nix

Reply:
I think that with `nixops` being specified in `default.nix`, we shouldn't need `nixops` in our deployment config anymore.

But that would take some work to implement -- should I add a checkbox in the PR merge checklist, @domenkozar?

Reply:
Also, this particular change is a revert of  _previous_ errorneous change -- apparently Nixops build isn't really deterministic, so on my system it yields a different store path -- with the same Nixpkgs.

If that even makes sense..

Reply:
That shouldn't happen if same nixpkgs commit is used.

Reply:
It shouldn't, yet it does.. : -/  Some debugging is in order..